src/app/components/SearchResultsPage/SearchResultsPage.jsx

```diff
@@ -69,7 +79,14 @@ class SearchResultsPage extends React.Component {
         title="Search Results | Shared Collection Catalog | NYPL"
       >
         <main className="main-page">
-          <LoadingLayer status={this.state.isLoading} title="Searching" />
+          {this.state.isLoading &&
```

Comment:
You have this check here but not in the other places where you are rendering `LoadingLayer`, why?

Reply:
Ah, I was testing it, but then I realized I don't need it. I probably didn't commit that change. I will remove it now.